css-colors/styles.css

```diff
@@ -0,0 +1,51 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+section {
+  padding: 1.5rem 0;
+  background-color: rgb(235 245 255);
+}
+
+h2 {
+  margin-bottom: 3.5rem;
+  font-size: 2rem;
+  text-align: center;
+  font-weight: 300;
+}
+
+form {
+  margin: 0 auto 2rem;
+  text-align: center;
+}
+
+input,
+button {
+  height: 3rem;
+  border-radius: 0.25rem;
+  font-size: 1.1rem;
+}
+
+input {
+  width: 400px;
+  padding: 0.5rem;
+  border: 1px solid;
+  font-family: Roboto, sans-serif;
+  font-weight: 300;
+  background-color: rgb(240 240 240);
```

Comment:
From the design specs we only want to style the input border. You can do this with the `border-color` property rather than `background-color` property.